app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/HomeScreen.kt

```diff
@@ -135,11 +142,29 @@ private fun HomeScreenContent(
                         )
                     }
                 } else {
-                    items(trendingCoins) { coin ->
+                    itemsIndexed(trendingCoins) { index, coin ->
+                        if (index >= trendingCoinsLastIndex) {
+                            SideEffect {
+                                Timber.d("onTrendingCoinsLoadMore at index: $index, lastIndex: $trendingCoinsLastIndex")
+                                onTrendingCoinsLoadMore?.invoke()
+                            }
+                        }
+
                         Box(modifier = Modifier.padding(start = Dp16, end = Dp16, bottom = Dp16)) {
                             TrendingItem(
                                 coinItem = coin,
-                                onItemClick = { onTrendingItemClick.invoke(coin) }
+                                onItemClick = { onTrendingItemClick?.invoke(coin) }
+                            )
+                        }
+                    }
+
+                    // Load more loading
+                    if (showTrendingCoinsLoading == LoadingState.LoadingMore) {
+                        item {
+                            CircularProgressIndicator(
+                                modifier = Modifier
+                                    .fillMaxWidth()
+                                    .wrapContentWidth(align = Alignment.CenterHorizontally),
```

Comment:
Would it be possible to add vertical padding for this loading indicator? 😄

Reply:
@hoangnguyen92dn there is no design for this loading indicator, I'm not sure about the size & alignment so I kept the default value. I think it's good enough, right?

<img width="552" alt="image" src="https://user-images.githubusercontent.com/16315358/200479631-92b0a1e5-18ea-4a5b-8e41-a2dbb49d0bff.png">

Reply:
@luongvo On my device it shows at the bottom like this,

<img src="https://user-images.githubusercontent.com/6950766/200505736-8e459bf4-b40b-4497-a7ad-5dfd3c3cde79.png" width=300 />

Reply:
@hoangnguyen92dn better now I believe

<img width="552" alt="image" src="https://user-images.githubusercontent.com/16315358/200509251-bbbba365-e3ea-4d0e-9f6f-3f586c405702.png">

Reply:
@luongvo Looks good now 🚀